lib/matplotlib/type1font.py

```diff
@@ -309,7 +309,7 @@ def suppress(tokens):
 
         while True:
             token, value = next(tokens)
-            if token == 'name' and value in table:
+            if token is cls._name and value in table:
```

Comment:
I introduced these sentinel objects to replace string comparisons, which were hard to get right across Python versions. The idea would be to use them like None. Perhaps this needs better comments, or one of the other enum patterns.